backend/src/main/java/com/constval/repook/service/ReportBookletCreationService.java

```diff
@@ -0,0 +1,18 @@
+package com.constval.repook.service;
+
+
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@CrossOrigin(origins = "http://localhost:8081")
+@RestController
+@RequestMapping("/api")
```

Comment:
würde hier `("/api/booklet")` oder so bevorziehen

Reply:
Wie schon erwähnt war dies als Proof of concept gedacht. Aber guter Vorschlag.